src/bsonjs.c

```diff
@@ -42,8 +43,20 @@ bson_str_to_json(const char *bson, size_t bson_len, size_t *json_len)
         bson_reader_destroy(reader);
         return NULL;
     }
-
-    json = bson_as_json(b, json_len);
+    if(strcmp(extended_str, "relaxed") == 0){
```

Comment:
What do you think about implementing this without string parsing? Instead we can provide module level constants:
```
LEGACY = 0
RELAXED = 1
CANONICAL = 2
```

The dumps can become:
```
dumps(..., mode=CANONICAL)
```
And we can just to regular int checks instead of strcmp.

Note to accomplish this you would use [PyModule_AddIntConstant](https://docs.python.org/3/c-api/module.html?highlight=pymodule_addintconstant#c.PyModule_AddIntConstant) similar to how we add the `"__version__"` attribute.

Reply:
Done.